qa/integration-tests/src/test/java/io/camunda/it/utils/MultiDbConfigurator.java

```diff
@@ -164,6 +164,45 @@ public void configureRDBMSSupport() {
         });
   }
 
+  public void configureAWSOSupport(final String opensearchUrl, final String indexPrefix) {
+    final OperateOpensearchProperties operateOpensearch = operateProperties.getOpensearch();
```

Comment:
❓ 🔧  I think we should simply reuse the other Opensearch configuration, can we call/delegate to the other and then afterwards set the was enabled?

Reply:
Reused